smooth/framework/functions/print_results.py

```diff
@@ -7,20 +7,33 @@ def print_smooth_results(smooth_results):
 
     # Calculate the sum of all total annuities [EUR/a].
     sum_of_tot_annuity = 0
+    sum_of_tot_emissions_annuity = 0
 
     print("\n++++++++")
     print('RESULTS:')
     print("++++++++\n")
-    print('{:20s} {:20s} {:20s} {:20s} {:20s}'.format(
-        'component name', 'annutiy capex', 'annuity opex', 'annuity var. cost', 'annuity total'))
+    print('{:20s} {:20s} {:20s} {:20s} {:20s} {:20s} {:20s} {:20s}'.format(
+        'component name', 'annutiy capex', 'annuity opex', 'annuity var. cost', 'annuity total',
+        'annutiy fix GGE', 'annuity var. GGE', 'annuity total GGE'
```

Comment:
What does GGE mean?

Reply:
greenhous gas emissions (GGE)
I was not sure wheather you want to consider CO_2 only or all emissions, but I thought in any case GGE would be the right term as it includes it